### ISSUE TITLE
tine does not start anymore

### ISSUE THREAD

**Issue by mokraemer** [tine20/tine20#7452]
version 2023.02.04.50 gives the following loop in log:
Tinebase-75773f365d5ce49fcc70-FAT.js:2 logger initialized
15:22:29.662 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:30.357 Tinebase-75773f365d5ce49fcc70-FAT.js:2 getSurveyPanel() - fetching survey data ...
15:22:30.392 DevTools failed to load source map: Could not parse content for https://tine.aldente-it.de/Tinebase/js/Tinebase-75773f365d5ce49fcc70-FAT.js.map: Expected ',' or ']' after array element in JSON at position 4424247
15:22:30.471 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:30.860 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:31.015 Tinebase-75773f365d5ce49fcc70-FAT.js:2 Ext.isObject.e {phantom: true, id: 'ext-record-1', data: {…}, json: {…}, store: constructor}data: {title: 'Tine 2.0 ActiveSync Umfrage', subtitle: 'Welche ActiveSync fähigen Geräte verwendest du und was funktioniert und was nicht?', duration: 2, langs: 'Deutsch', link: 'https://docs.google.com/forms/d/e/1FAIpQLScPllT7QBeZqx8w1LL-4f0amFiPiJeB1y00BKHT80id6e1zcQ/viewform', …}id: "ext-record-1"json: {title: 'Tine 2.0 ActiveSync Umfrage', subtitle: 'Welche ActiveSync fähigen Geräte verwendest du und was funktioniert und was nicht?', duration: 2, langs: 'Deutsch', link: 'https://docs.google.com/forms/d/e/1FAIpQLScPllT7QBeZqx8w1LL-4f0amFiPiJeB1y00BKHT80id6e1zcQ/viewform', …}phantom: truestore: constructor {data: Ext.u…l.MixedCollection, removed: Array(0), proxy: Ext.d…a.ScriptTagProxy, reader: Ext.d…a.JsonReader, baseParams: {…}, …}[[Prototype]]: Ext.data.Record
15:22:31.020 Tinebase-75773f365d5ce49fcc70-FAT.js:2 Survey version: 2017.08.11 / Tine version: 2023.02.04.50
15:22:31.021 Tinebase-75773f365d5ce49fcc70-FAT.js:2 Survey enddate: Thu Feb 08 2018 00:00:00 GMT+0100 (Mitteleuropäische Normalzeit)
15:22:31.384 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:31.570 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:32.252 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:32.434 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:32.640 DevTools failed to load source map: Could not parse content for https://tine.aldente-it.de/Tinebase/js/Tinebase-75773f365d5ce49fcc70-FAT.js.map: Expected ',' or ']' after array element in JSON at position 4424247
15:22:33.009 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:33.179 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:33.815 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:33.982 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:34.518 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:34.703 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
15:22:35.338 Tinebase-75773f365d5ce49fcc70-FAT.js:2 renderWindow::start
15:22:35.516 Tinebase-75773f365d5ce49fcc70-FAT.js:2 tineInit::clearRegistry
[...]

**Comment by lab-at-nohl** [comment]
Good to know. Do you clear session files after updates?

**Comment by mokraemer** [comment]
No. I didn't know the changes of the version affects stored session information.
